.github/workflows/diff-report.yml

```diff
@@ -97,6 +97,36 @@ jobs:
           # shellcheck disable=SC2005
           echo "$(cat "$GITHUB_OUTPUT")"
 
+      - name: Extract config name
+        id: extract_config_name
+        run: |
+          CONFIG_NAME=$(grep -oP '(?<=Diff Regression config: ).*?(?=/config.xml)' .ci-temp/text | awk -F'/' '{print $(NF-1)"/"$NF}')
+          echo "config_name=$CONFIG_NAME" >> $GITHUB_OUTPUT
+          echo "Extracted config name: $CONFIG_NAME"
```

Comment:
what for we have this block?
we already print all variables where links are stored.

Reply:
So, whenever a user is trying to put configs in PR description, i am extrating the modulename and comparing with the github comment, if they match, only proceed otherwise the report fails.

Reply:
We don't need this at all.
Let's skip handling any content from PR description for now